code/modules/religion/religion_sects.dm

```diff
@@ -216,3 +216,31 @@
 	adjust_favor(20, user) //it's not a lot but hey there's a pacifist favor option at least
 	qdel(offering)
 	return TRUE
+
+/**** Carp Sect ****/
+
+/datum/religion_sect/carp_sect
+	name = "Followers of the Great Carp"
+	desc = "A sect dedicated to the space carp and carp'sie."
+	convert_opener = "Drown the station in fish and water.<br>Sacrificing meat grants you favor."
+	alignment = ALIGNMENT_NEUT
+	max_favor = 10000
+	desired_items = list(/obj/item/reagent_containers/food/snacks/meat)
+	rites_list = list(/datum/religion_rites/summon_carp, /datum/religion_rites/flood_area, /datum/religion_rites/summon_carpsuit)
+	altar_icon_state = "convertaltar-blue"
+
+//Carp bibles give people the carp faction!
+/datum/religion_sect/carp_sect/sect_bless(mob/living/blessed, mob/living/user)
+	if(isliving(blessed))
```

Comment:
Early return here instead.

Reply:
Slightly confused as to what you mean here

Reply:
adjusting this completely, so it has the regular heal as well as the carp-blessing